composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "victoire/victoire": "~2.3 | ~3.0",
+        "victoire/victoire": "~2.2 | ~2.3 | ~3.0",
```

Comment:
I don't understand what you're trying to do here as `~2.2` means `>=2.2.0.0-dev <3.0.0.0-dev` https://getcomposer.org/doc/articles/versions.md#tilde-version-range-